polaris.shopify.com/src/pages/contributing/[doc].tsx

```diff
@@ -21,15 +20,13 @@ const contributingDirectory = path.join(process.cwd(), "content/contributing");
 
 const Contributing: NextPage<Props> = ({ readme, title }: Props) => {
   return (
-    <Container>
```

Comment:
@chloerice Sorry that this isn't better documented yet, but `Layout` doesn't need a `Container`, so I removed this and re-added the padding around `Layout`.